KarhooUISDK/Screens/CheckoutScreen/NewCheckout/Workers/KarhooNewCheckoutBookingWorker.swift

```diff
@@ -33,7 +33,7 @@ final class KarhooNewCheckoutBookingWorker: NewCheckoutBookingWorker {
     private let userService: UserService
     private let tripService: TripService
     private let sdkConfiguration: KarhooUISDKConfiguration
-    private let paymentWorker: KarhooNewCheckoutPaymentWorker
+    private let paymentWorker: NewCheckoutPaymentWorker
```

Comment:
`NewCheckoutPaymentWorker` is a protocol name, while `KarhooNewCheckoutPaymentWorker` is a name of the class.

Reply:
Then can we have `CheckoutPaymentWorker` for the protocol and `KarhooCheckoutPaymentWorker` for the class? I'd really like to avoid having "new" in any type name

Reply:
Sure, no problem

Reply:
As long as we clean up at the end, then it's perfectly fine to leave it now.